040_Report_L337.php

```
@@ -0,0 +1,772 @@
+                $defVal = $ans != '' ? $ans : $this->arrayVariable[$macthV]['initialValue'];
+                $out['target'] = str_ireplace(array('$V{' . $macthV . '}'), array($ans), $out['target']);
+            }
+        }
+        preg_match_all("/F{(\w+)}/", $out['target'], $matchesF);
+        if ($matchesF) {
+            foreach ($matchesF[1] as $macthF) {
+                $out['target'] = $this->getValOfField($macthF, $row, $out['target']); //str_ireplace(array('$F{'.$macthF.'}'),array(utf8_encode($row->$macthF)),$out['target']); 
+            }
+        }
+        $htmlData = array_key_exists('htmlData', $this->arrayVariable) ? $this->arrayVariable['htmlData']['class'] : '';
+        if (preg_match('/(\d+)(?:\s*)([\+\-\*\/])(?:\s*)/', $out['target'], $matchesMath) > 0 && $htmlData != 'HTMLDATA') {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$matchesMath'. (undefined)

(UnusedLocalVariable)